notebooks/7-rotation-speeds/interm_rotation_rate_solutions.ipynb

```diff
@@ -0,0 +1,3039 @@
+{
```

Comment:
<p>Can we plot this separately, or perhaps in the other order (black first and thus "on top")? I think it's compelling to see the residual peaks in the black plot</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>